Instructions-for-Reviewers.md

```diff
@@ -35,7 +35,8 @@ Simply visit this URL: https://github.com/pulls/assigned. It takes you to a list
      * **Important:** Please also ensure that, if a PR author files a TODO for future work, they assign themselves to it (or arrange for it to be assigned responsibly to someone else) and don't just abandon it. It should typically be the next thing they work on, unless there is a principled reason to agree otherwise. (In other words, PR authors shouldn't just file TODOs for future work and then abandon them.)
 
 
-      **Note**: To checkout branch BRANCH_NAME from committer COMMITTER_USERNAME to your local machine, run:
+      > **Note**
+      > To checkout branch BRANCH_NAME from committer COMMITTER_USERNAME to your local machine, run:
       ```
```

Comment:
I think the lines below also go inside the note, right?

Reply:
yes, you are right. Updated,